teuthology/test/test_provision.py

```diff
@@ -0,0 +1,54 @@
+from mock import Mock, MagicMock
+
+from teuthology import provision
+
+
+class TestDownburst(object):
+    def test_create_if_vm(self):
+        ctx = Mock()
```

Comment:
The creation of ctx should probably be in a setup method if it's going to be the same across the tests.